tabbycat/notifications/admin.py

```diff
@@ -1,12 +1,18 @@
+from typing import Any, Callable, Type, TYPE_CHECKING
+
 from django.contrib import admin
 from django.utils import timezone
 
 from utils.admin import ModelAdmin, TabbycatModelAdminFieldsMixin
 
 from .models import BulkNotification, EmailStatus, SentMessage
 
+if TYPE_CHECKING:
+    from django.db.models import QuerySet
+    from django.http.request import HttpRequest
+
 
-def precise_timestamp_isoformat(model, field_name):
+def precise_timestamp_isoformat(model: Type, field_name: str) -> Callable[[Any], str]:
```

Comment:
Would the type for `model` be a `Model`? Unsure about using `Type`. Also, `obj` from line 17 would be a model object, so why `Any`?

Reply:
My reasoning is that `precise_timestamp_isoformat` doesn't take a specific model instance but rather a type itself and then spits out a function based on that. For example, on line 44 it gets called like this: `precise_timestamp_isoformat(BulkNotification, 'timestamp')`, with the `BulkNotification` class, not an instance of `BulkNotification`.